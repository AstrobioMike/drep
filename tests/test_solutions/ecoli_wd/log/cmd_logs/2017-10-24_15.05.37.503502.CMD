/usr/local/bin/mash paste /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/MASH_files/ALL.msh /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/MASH_files/sketches/Enterococcus_casseliflavus_EC20.fasta.msh /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/MASH_files/sketches/Enterococcus_faecalis_T2.fna.msh /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/MASH_files/sketches/Enterococcus_faecalis_TX0104.fa.msh /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/MASH_files/sketches/Enterococcus_faecalis_YI6-1.fna.msh /Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/MASH_files/sketches/Escherichia_coli_Sakai.fna.msh
