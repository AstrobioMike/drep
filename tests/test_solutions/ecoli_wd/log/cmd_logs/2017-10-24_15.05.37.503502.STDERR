ERROR: "/Users/mattolm/Programs/drep/tests/test_backend/ecoli_wd/data/MASH_files/ALL.msh" exists; remove to write.
